### PR TITLE
Removed inline css to external css file

### PR DESCRIPTION
**Pull request by YoutLone**
# Hello Ama @khinezarthwe 👋👋 :

### In this PR I just removed inline CSS to external CSS file:

- [ ] Issue Link :  [https://github.com/khinezarthwe/mmdt_web_app/issues/84](url)

- [ ] Changed the css file name `paginator.css` to `custom.css` because of when I use the `paginator.css` at that time some of the css code is doesn't work properly but when I change the file name `custom.css` everything is working properly.

### CODE REVIEW

File: mmdt/mmdt/settings.py [hunk 1]
```diff
@@ -12,20 +12,24 @@
...
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config("SECRET_KEY")
+SECRET_KEY = 'django-insecure--ld-c!55j+l1gr@alfh&w#+flyd@r*y^wy_r+sokxv&i6d0%tg'
```
Comment (khinezar-mercari) [inline mmdt/mmdt/settings.py:24]: please put in your .env file then , you do not need to update in here 
can check the readme.md file.

File: mmdt/mmdt/settings.py [hunk 2]
```diff
@@ -12,20 +12,24 @@
...
+
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', cast=bool)
+ENVIRONMENT = os.getenv('ENVIRONMENT', 'development')
+
+if ENVIRONMENT == 'production':
+    DEBUG = config('DEBUG', default=False, cast=bool)
```
Comment (khinezar-mercari) [inline mmdt/mmdt/settings.py:30]: same in debug case, add DEBUG key in .env file , then you don't need to update in here. 
all are mentioned in readme.md file

File: mmdt/mmdt/settings.py [hunk 3]
```diff
@@ -151,24 +155,20 @@
...
-if AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY and  AWS_STORAGE_BUCKET_NAME:
-    #AWS S3 settings
+# Use local file system storage for development
+if DEBUG:
+    DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+else:
+    # Use S3 storage for production
+    AWS_ACCESS_KEY_ID = config('AWS_ACCESS_KEY_ID')
+    AWS_SECRET_ACCESS_KEY = config('AWS_SECRET_ACCESS_KEY')
+    AWS_STORAGE_BUCKET_NAME = config('AWS_STORAGE_BUCKET_NAME')
```
Comment (khinezar-mercari) [inline mmdt/mmdt/settings.py:165]: same with above comment.

File: mmdt/mmdt/settings.py [hunk 4]
```diff
@@ -151,24 +155,20 @@
...
     AWS_S3_SIGNATURE_NAME = "s3v4"
     AWS_S3_REGION_NAME = "us-west-1"
     AWS_S3_FILE_OVERWRITE = False
     AWS_DEFAULT_ACL = None
     AWS_S3_VERITY = True
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
-else:
-    #Local storage settings
-    MEDIA_URL = '/media/'
-    MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
```
Comment (khinezar-mercari) [inline mmdt/mmdt/settings.py:171]: why you remove this ? I think @Pyae24 added this block

File: mmdt/mmdt/settings.py [hunk 5]
```diff
@@ -177,27 +177,24 @@
...
-            'level': 'INFO',
-            'propagate': True,
+    'handlers': {
+        'rotate_file': {
+            'level': 'DEBUG',
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': os.path.join(log_dir, 'django.log'),
+            'maxBytes': 10 * 1024 * 1024,
+            'backupCount': 5,
+            'formatter': 'verbose',
```
Comment (khinezar-mercari) [inline mmdt/mmdt/settings.py:193]: I think that are not your change, please be sure you create the branch from main branch. We have already update those in main branch

File: mmdt/mmdt/settings.py (hunk 1 above)
Comment (YoutLone) [reply mmdt/mmdt/settings.py:24]: Sorry ama , I forgot to change that.

**Comment by khinezarthwe** [comment]
shall we update in other pages too.

Screenshot 2024-02-19 at 7 48 25 PM

**Comment by YoutLone** [comment]
Yes, I will also update for another inline css.

**Pull request merged**